WcaOnRails/db/migrate/20150504163657_create_doorkeeper_tables.rb

```diff
@@ -2,38 +2,38 @@
 class CreateDoorkeeperTables < ActiveRecord::Migration
   def change
     create_table :oauth_applications do |t|
-      t.string  :name,         null: false
-      t.string  :uid,          null: false
-      t.string  :secret,       null: false
-      t.text    :redirect_uri, null: false
-      t.string  :scopes,       null: false, default: ''
+      t.string :name, null: false
+      t.string :uid, null: false
+      t.string :secret, null: false
+      t.text :redirect_uri, null: false
+      t.string :scopes, null: false, default: ''
```

Comment:
This is kind of an interesting one. Rubocop actually *does* allow aligned code. However, this is in tension with their `Style/ExtraSpacing` rule, because without context, column aligned code looks like it has extra spaces. So I think what rubocop does is ignore the `Style/ExtraSpacing` rule if the next or previous lines have been spaced to align with the current line. This is problematic with the following code:

```ruby
 28     create_table :oauth_access_tokens do |t|
 29       t.integer  :resource_owner_id
 30       t.integer  :application_id
 31       t.string   :token,             null: false
 32       t.string   :refresh_token
 33       t.integer  :expires_in
 34       t.datetime :revoked_at
 35       t.datetime :created_at,        null: false
 36       t.string   :scopes
 37     end
```

```
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> bundle exec rubocop db/migrate/20150504163657_create_doorkeeper_tables.rb
Inspecting 1 file
C

Offenses:

db/migrate/20150504163657_create_doorkeeper_tables.rb:31:25: C: Unnecessary spacing detected.
      t.string   :token,             null: false
                        ^^^^^^^^^^^^
db/migrate/20150504163657_create_doorkeeper_tables.rb:35:30: C: Unnecessary spacing detected.
      t.datetime :created_at,        null: false
                             ^^^^^^^

1 file inspected, 2 offenses detected
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> 
```

However, if I reorder the lines with the `null: false` so they're touching, then the cop is happy:

```
 28     create_table :oauth_access_tokens do |t|
 29       t.integer  :resource_owner_id
 30       t.integer  :application_id
 31       t.string   :refresh_token      
 32       t.integer  :expires_in
 33       t.datetime :revoked_at
 34       t.datetime :created_at,        null: false
 35       t.string   :token,             null: false
 36       t.string   :scopes
 37     end
```

```
~/gitting/worldcubeassociation.org/WcaOnRails @kaladin> bundle exec rubocop db/migrate/20150504163657_create_doorkeeper_tables.rb
Inspecting 1 file
.

1 file inspected, no offenses detected
```

I've opted to reorder the lines here, rather than turning off the extra spacing cop.